meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -60,7 +61,10 @@ FILES_${PN}-lite = " \
                     ${nonarch_libdir}/tmpfiles.d/${PN}-lite.conf \
                     "
 FILES_${PN}-lite-lib = "${nonarch_libdir}/lib${PN}_lite.so"
-FILES_${PN}-lite-dev = "${includedir}/${PN}-lite"
+FILES_${PN}-lite-dev = " \
+                        ${includedir}/${PN}-lite \
+                        ${libdir}/pkgconfig/aktualizr.pc \
```

Comment:
Shouldn't this pc file be part of aktualizr-dev instead?

Reply:
I think so, let me check if it works.

Reply:
It requires adding of the libaktualizr's header files too,

```
+FILES_${PN}-dev = " \
+                   ${libdir}/pkgconfig/aktualizr.pc \
+                   ${includedir}/libaktualizr \
+                  "
```

Reply:
Tested https://ci.foundries.io/projects/msul-dev01/lmp/builds/1084/.

Reply:
BTW, is there any way to specify cmake target for a specific package (e.g. sota-tools). I came across this flag `OECMAKE_TARGET_COMPILE`, which allows specifying a desired target to build, but how to set it just for `sota-tools` package?